src/pages/playlist/playlist card/PlaylistCard.jsx

```diff
@@ -0,0 +1,24 @@
+import { useNavigate } from 'react-router-dom'
+import { usePlaylist } from '../../../context/playlist-context';
+import classes from './playlistCard.module.css'
+
+const PlaylistCard = ({list}) => {
+    const navigate = useNavigate();
+    const {removePlaylist} = usePlaylist();
+    const closeHandler=async (e)=>{
+        try{
+            e.stopPropagation();
```

Comment:
Could you tell me why it is used? I searched it online but could not get the proper usecase of it.